README.rst

```diff
@@ -692,6 +693,8 @@ to the default list of symfony-level fixers:
         ->finder($finder)
     ;
 
+NOTE: `exclude` will work only for directories, so if you need to exclude file, try `notPath`.
```

Comment:
No. We do not have (and do not want to have) our own finder.
For us finder is anything that will implement `\Traversable`.

symfony/finder is only example